layouts/shortcodes/checkin-schedule.html

```diff
@@ -29,6 +29,7 @@
         "Parallel Rustc",
         "Polonius",
         "Polymorphization",
+        "Prioritization",
```

Comment:
Do we need/want check-ins from prioritization wg?

Reply:
I had the same doubt but ended thinking well if there's nothing to say we can skip the checkin but there may be changes to share like ... we've changed labels in this or that way, we are doing our prioritization async, at the beginning we had 50 P-high issues we now have 10 and stuff like that :). Probably a lot of things could happen at the beginning I guess long term we should remove this from here. Still not 100% sure, let me know what you think.

Reply:
This is the right thing to do per the working group formation guidelines but I wonder if perhaps we should consider changing that. Adding this wg will bring us to 13 wgs which means, if I've done my math correctly, we'll only hear from a wg every 1.5 months. That seems like a fairly long time to find out if a wg is stalled or blocked on something and not making progress (especially if that wg is doing work that's part of our yearly goals). 

Perhaps we should consider only requiring checkins from teams working toward our yearly goals? Other teams can of course make use of the announcement time at the beginning of the weekly compiler team meeting to report updates on anything happening. 

(I don't feel strongly about this, it's just an idle thought I had.)

Reply:
I have no clear idea either. Another possibility is to do 3 checkins per week. Mainly given that from time to time there are some wgs with no updates to share.

Reply:
I think let's merge as is and discuss changing how we handle check-ins later. I also don't actually think every 1.5 months is that bad, but I do think we can do better.